check.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Find all .qnt files
+qnt_files=$(find examples -type f -name "*.qnt")
+
+# Initialize a flag to track if any file fails the check
+all_files_valid=true
+
+# Loop through each .qnt file and check its header
+for file in $qnt_files; do
+  if [[ "$(head -n 1 "$file")" != "// -*- mode: Bluespec; -*-" ]]; then
+    echo "File $file does not start with // -*- mode: Bluespec; -*-"
```

Comment:
Great idea! This will be useful :smile:

Reply:
Woops forgot to remove this, wanted to do this in a GitHub Actions workflow. Will open a separate PR for it.

Reply:
25de4982